pepper/cli.py

```diff
@@ -108,6 +108,13 @@ def parse(self):
             ''')
         )
 
+        self.parser.add_option(
+            '--output-force-color', dest='output_force_color', default=False, action='store_true',
+            help=textwrap.dedent('''
+                Force salt outputter to use color when printing.
+            ''')
```

Comment:
I'd like to have options match the `salt` command where possible:

```bash
    --force-color, --force-colour
                        Force colored output.
```

Would you be OK with updating that to support `--force-color` and `--force-colour`? 
If so, probably best to also update the var to `force_color`